Source/WebCore/platform/graphics/gstreamer/GStreamerRegistryScanner.cpp

```diff
@@ -1109,15 +1109,13 @@ void GStreamerRegistryScanner::fillVideoRtpCapabilities(Configuration configurat
                         continue;
                 } else {
                     auto spsAsInteger = parseInteger<uint64_t>(profileLevelId, 16).value_or(0);
-                    WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // GLib port
-                    uint8_t sps[3];
+                    std::array<uint8_t, 3> sps;
                     sps[0] = spsAsInteger >> 16;
                     sps[1] = (spsAsInteger >> 8) & 0xff;
                     sps[2] = spsAsInteger & 0xff;
```

Comment:
You may want to change this into an initializer list before landing:

```suggestion
                    std::array<uint8_t, 3> sps { spsAsInteger >> 16, (spsAsInteger >> 8) & 0xff, spsAsInteger & 0xff };
```

Reply:
This isn't working, would need some casts maybe... I'll revert this part.

Reply:
Yeah, I wasn't sure it would work directly, that's why I had approved the PR already. I *think* assignment works because the compiler will truncate the upper bits in that case, but for initializer lists only widening is allowed. Feel free to leave the assignments if you feel that the code is clearer that way instead of adding casts around.